### ISSUE TITLE
Exporting SQL query history

### ISSUE THREAD

**Issue by pawelsalawa** [pawelsalawa/sqlitestudio#4286]
I would be happy with an export feature to export all my queries history to a text file - or a csv file export giving the fields you currently hold/display

**Comment by NirmalSilwal** [comment]
Is this feature available now in sqllitestudio? @pawelsalawa

**Comment by pawelsalawa** [comment]
No, it's still in Open status and the Milestone assigned is 3.5.0 (at least at the moment).

**Comment by pawelsalawa** [comment]
Function is now implemented. Not closing the issue, as the icon for history export should be specialized. This is a last thing left to do.
